versions/3.0.md

```diff
@@ -741,7 +740,7 @@ A unique parameter is defined by a combination of a [name](#parameterName) and [
 There are four possible parameter types.
 * Path - Used together with [Path Templating](#pathTemplating), where the parameter value is actually part of the operation's URL. This does not include the host or base path of the API. For example, in `/items/{itemId}`, the path parameter is `itemId`.
 * Query - Parameters that are appended to the URL. For example, in `/items?id=###`, the query parameter is `id`.
-* Header - Custom headers that are expected as part of the request.
+* Header - Custom headers that are expected as part of the request. Note that [RFC 7230](https://tools.ietf.org/html/rfc7230#page-22) states header names are case insensitive.
```

Comment:
For consistency, should we wrap the _"Note"_ in parentheses like in other places?